server/src/test/java/org/elasticsearch/index/mapper/ContentPathTests.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.index.mapper;
+
+import org.elasticsearch.test.ESTestCase;
+
+public class ContentPathTests extends ESTestCase {
+
+    public void testAddPath() {
+        ContentPath contentPath = new ContentPath();
+        contentPath.add("foo");
+        contentPath.add("bar");
+        contentPath.add("baz");
+        String[] path = contentPath.getPath();
```

Comment:
I would prefer that all these tests that call `.getPath()` instead call `.pathAsText()`

The only exception would be `public void testRemovePath() {` that it check for `null` AND that it calls `.pathAsText()`

Reply:
You are right, good catch! I changed all the affected tests. 
Thanks for the review!